src/engine/subscription/Subscription.spec.ts

```diff
@@ -0,0 +1,575 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+
+import {
+  Subscription,
+  isSubscription,
+  SUBSCRIPTION_TYPE_CREATE,
+  SUBSCRIPTION_TYPE_UPDATE,
+  SUBSCRIPTION_TYPE_DELETE,
+  processEntitySubscriptions,
+  pubsub,
+} from './Subscription';
+import { Entity } from '../entity/Entity';
+import { DataTypeString } from '../datatype/dataTypes';
+import { passOrThrow } from '../util';
+import { generateTestSchema } from '../../graphqlProtocol/test-helper';
+import { generateGraphQLSchema } from '../../graphqlProtocol/generator';
+import { subscribe, parse } from 'graphql';
+
+describe('Subscription', () => {
```

Comment:
Most of the tests here are inspired by Mutation test, differences resides in preProcessor, postProcessor below.

Reply:
yes, makes sense